docs/en/advanced-concepts/configuration-system.md

```diff
@@ -271,3 +271,5 @@ Keywords are defined in `lib/schema/keywords.js`.
 - `appiumCliDest`: allows a schema to specify a custom property name in the post-`argprase` arguments objects. If not set, this becomes a camelCased string.
 - `appiumCliDescription`: allows a schema to override the description of the argument when displayed on the command-line. This is useful paired with `appiumCliTransformer` (or `array`/`object`-typed properties), since there's a substantial difference between what a CLI-using user can provide vs. what a config-file-using user can provide.
 - `appiumCliTransformer`: currently a choice between `csv` and `json`. These are custom functions which post-process a value. They are not used when loading & validating config files, but the idea should be that they result in the same object you'd get if you used whatever the config file wanted (e.g., an array of strings). `csv` is for comma-delimited strings and CSV files; `json` is for raw JSON strings and `.json` files.
+- `appiumCliIgnore`: If `true`, do not support this property on the CLI.
```

Comment:
I usually add `is` to better reflect a variable is of boolean type

Reply:
I agree, but in this case, it's not a variable--it's a schema keyword.  the convention doesn't use "is", and we also want to be sure we are namespacing the keyword.

but could do `appiumShouldCliIgnore` or `appiumCliShouldIgnore` or something as @jlipps suggests

Reply:
@jlipps I changed it to `appiumCliIgnored`, which is more obviously a boolean (precedent: the `deprecated` keyword is a boolean, and uses the same tense).  That work?

Reply:
yep!